modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -383,6 +397,76 @@ const EditFDSFieldModalContent = ({
 	const fdsFieldLabelInputId = `${namespace}fdsFieldLabelInput`;
 	const fdsFieldRendererSelectId = `${namespace}fdsFieldRendererSelectId`;
 
+	const options = FDS_INTERNAL_CELL_RENDERERS.map(
+		(renderer: FDSInternalCellRenderer) => ({
+			label: renderer.label!,
+			value: renderer.name!,
+		})
+	);
+
+	options.push(
+		...fdsClientExtensionCellRenderers.map((item) => ({
+			label: item.name!,
+			value: item.erc!,
+		}))
+	);
+
+	const CellRendererDropdown = ({
+		cellRenderers,
+		namespace,
+		onItemClick,
+	}: {
+		cellRenderers: {
+			label: string;
+			value: string;
+		}[];
+		namespace: string;
+		onItemClick: Function;
+	}) => {
+		const cellRendererERCs = fdsClientExtensionCellRenderers.map(
+			(cellRendererCET) => cellRendererCET.erc
+		);
```

Comment:
[non-blocking nit] This should be named to contain CE, maybe `fdsClientExtensionCellRenderersERCs`

Reply:
Done!